make-release.sh

```diff
@@ -158,10 +158,7 @@ sed_in_place -r -e "/@eclipse-che\/api|@eclipse-che\/workspace-client|@eclipse-c
 npm --no-git-tag-version version --allow-same-version "${VERSION}"
 popd >/dev/null || exit
 
-echo "Copying source code to dockerfile directory"
-cp -r "tests/e2e" "dockerfiles/e2e/e2e"
-
-docker build -t quay.io/eclipse/che-e2e:${VERSION} dockerfiles/e2e/
+docker build -t quay.io/eclipse/che-e2e:${VERSION} -f /build/dockerfiles/Dockerfile . 
```

Comment:
`/build`? is this running inside a Dockerfile context?

Reply:
should this be `tests/e2e/build/dockerfiles/Dockerfile` ?

Reply:
and also all the files you want are not in `.` but in tests/e2e/

Reply:
we did a pushd to tests/e2e eralier, so the path should be good, without the first slash?)